modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -9,47 +9,46 @@ A message type in Partner Manager can be used in one of the following ways:
+Message types are always managed in the host profile page, regardless of whether they are used for a single partner or for multiple trading partners. Additionally, message types are reusable across message flows for different trading partners.
 
-Create message types from the Host page or from the message flow configuration wizard. When you create message types during message flow configuration, Partner Manager automatically designates the usage for the message type.
+You can create message types from the Host page or from the Message Flow wizard. When you create message types during message flow configuration, Partner Manager automatically designates the usage for the message type.
 
 == Create a Message Type
 
-. In Partner Manager left navigation, select *<host-name>*. 
+. In Partner Manager left navigation, select *<host-name>*.
 . From the *Message types* section host page:
-. Click *New*.
-. In the *Usage* drop-down list, select the message type use:
+.. Click *New*.
+..  In the *Usage* drop-down list, select the message type:
 * <<receive-and-send-to-partners,*Receive from Partners*>>
 * <<target-at-host,*Target at host*>>
 * <<source-at-host,*Source at host*>>
 * <<receive-and-send-to-partners,*Send to partners*>>
 +
-If you create a message type from the message flow, Partner Manager automatically designates the *Usage* value. 
-. Configure the message type according to the usage you selected.
+If you create a message type from the message flow, Partner Manager automatically determines the *Usage* value.
 . Click *Save*. +
 The message type appears in the *Message types* section on the host page.
 
 [[receive-and-send-to-partners]]
 == Receive From and Send To Partners
 
-These are inbound transactions that are received from your trading partners or outbound transactions your organization sends to your trading partners, typically following EDI standards such as X12.
+Inbound transactions that are received from your trading partners or outbound transactions that your organization sends to your trading partners typically follow EDI standards such as X12.
 
 Use the following settings to define X12 as an inbound or outbound message type:
 
 [%header%autowidth.spread]
 |===
 |Setting |Description |Default | Required
-|Format | Format of the message, which is `X12`. |`X12`|Yes
+|Format | The format of the message. |`X12`|Yes
 |Version | Select a version from the drop-down list. Valid values include: `3010`, `3020`, `3030`, `3040`, `3050`, `3060`, `3070`, `4010`, `4020`, `4030`, `4040`, `4050`, `4060`, `5010`, `5020`, `5030`, `5040`, `5050`, `6020`, `6030`, `6040`, `6050`, `7010`, `7020`, `7030`, `7040`, `7050`
  | None |Yes
-|Message Type |This is the X12 transaction set number, such as `850` for Purchase Order or `214` for Shipment Status. |None | Yes
-|Schema a| Options include: 
+|Message Type |The X12 transaction set number, such as `850` for Purchase Order or `214` for Shipment Status. |None | Yes
```

Comment:
If Purchase Order and Shipment Status are UI labels, then they should be in bold.

Reply:
Done